dashboard/app/controllers/rubrics_controller.rb

```diff
@@ -118,7 +118,8 @@ def run_ai_evaluations_for_user
 
     script_level = @rubric.lesson.script_levels.find {|sl| sl.levels.include?(@rubric.level)}
 
-    is_ai_experiment_enabled = current_user && Experiment.enabled?(user: current_user, script: script_level.script, experiment_name: 'ai-rubrics')
+    is_ai_experiment_enabled = true
+    #current_user && Experiment.enabled?(user: current_user, script: script_level.script, experiment_name: 'ai-rubrics')
```

Comment:
Consider removing this comment?

Reply:
That's actually an error I need to fix! I commented that line out to enable ai analysis locally for testing.